API.md

```diff
@@ -0,0 +1,66 @@
+# Octofet API
+
+## `class Octofet`
+
+Uses: class Spi2Parallel
+
+Creates an object of type `Octofet` to communicate with a particular [Octofet board](https://my.amperka.com/modules/octofet).
+
+One Octofet board includes 8 power switches. The following conventions are used: `pin` is a single power switch and `port` is a single Octofet device.
+
+### `Octofet(uint8_t pinCS, uint8_t devicesCount = 1)`
+
+Constructs a new object of the data bus hardware SPI with arguments:
+
+- `pinCS`: pin Chip Select is used to control the shift-register latch. Any GPIO-pin can be used.
+- `devicesCount`: quantity of the Octofet boards connected in a chain: the logical output pin of one module is connected to the input of another. If the `devicesCount` argument is omitted, it controls only one Octofet board.
```

Comment:
Just above you say “pin is a single power switch” and “port is a single Octofet device”. So… `pinCS` is a power switch used to control a latch. And logical power switches of one module are connected to an input of another :astonished: 

Sounds like a mess.